doc/api/deprecations.md

```diff
@@ -660,6 +660,15 @@ Type: Runtime
 
 `REPLServer.parseREPLKeyword()` was removed from userland visibility.
 
+<a id="DEP00XX"></a>
+### DEP00XX: path._makeLong()
+
+Type: Runtime
```

Comment:
@jasnell I’m okay with that, I guess (not a fan, but eh).

Also, if this is just a docs-deprecation, I’d be in favour of just landing it as a minor in v8.x.